main.js

```diff
@@ -25,3 +25,135 @@ contactForm.addEventListener('submit', (event) => {
+const year = document.querySelector('.year');
+
+const projectsList = [
+  {
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    imageLink: 'images/Snapshoot2.png',
+    technologies: ['html', 'css', 'javascript'],
+    role: 'Back end dev',
+    company: 'canopy',
+    year: '2015',
+    sourceLink: 'https://github.com/thermiee/Portofolio/',
+    demoLink: 'https://thermiee.github.io/Portofolio/',
+  },
+  {
+    name: 'Multi-post stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    imageLink: 'images/Snapshoot.png',
+    technologies: ['html', 'css', 'javascript'],
+    role: 'Full Stack Dev',
+    company: 'Facebook',
+    year: '2015',
+    sourceLink: 'https://github.com/thermiee/Portofolio/',
+    demoLink: 'https://thermiee.github.io/Portofolio/',
+  },
+  {
+    name: 'Facebook 360',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. ',
+    imageLink: 'images/Snapshoot1.png',
+    technologies: ['html', 'css', 'javascript'],
+    role: 'Back end Dev',
+    company: 'Facebook',
+    year: '2015',
+    sourceLink: 'https://github.com/thermiee/Portofolio/',
+    demoLink: 'https://thermiee.github.io/Portofolio/',
+  },
+  {
+    name: 'Uber Navigation',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    imageLink: 'images/Snapshoot3.png',
+    technologies: ['html', 'css', 'javascript'],
+    role: 'Lead developer',
+    company: 'Uber',
+    year: '2018',
+    sourceLink: 'https://github.com/thermiee/Portofolio/',
+    demoLink: 'https://thermiee.github.io/Portofolio/',
+  },
+];
+
+workSections.innerHTML = '';
+for (let i = 0; i < projectsList.length; i += 1) {
+  let languages = '';
+  projectsList[i].technologies.forEach((language) => {
+    languages += `
+      <li><a href='' class='lang1' title='${language}' class='lang-2'>${language}</a></li>
+    `;
+  });
+  workSections.innerHTML += `
+      <div class='works'>
+          <div class='works__image'>
+            <img src='${projectsList[i].imageLink}' alt='work 1' class='img1' />
+          </div>
+          <div class='work__description'>
+            <h2>${projectsList[i].name}</h2>
+            <div class='short_desc'>
+              <h4>${projectsList[i].company}</h4>
+              <span></span>
+              <h4>${projectsList[i].role}</h4>
+              <span></span>
+              <p>${projectsList[i].year}</p>
+            </div>
+            <div class='long_desc'>
+              <p>
+                ${projectsList[i].description}
+              </p>
+
+              <ul class='work__lang'>
+                ${languages}
+              </ul>
+            </div>
+            <button type='button' class='btn btn-enabled project-modal-${i}'>See project</button>
+          </div>
```

Comment:
Hello @team, you have done a great job on this project and you have implemented the popup window quite well 💪🏻! However, I have noticed that the desktop view of the popup does not render quite well, the content is not all in view yet the user cannot scroll to see it. When I scroll its the background content that scrolls - though it looks good on the mobile view. I request that you make changes to address this issue.

https://user-images.githubusercontent.com/4247740/151641509-d3ce0835-18a3-4df9-8197-c6031dcda72c.mp4